speech/setup.py

```diff
@@ -50,14 +50,14 @@
 }
 
 REQUIREMENTS = [
-    'google-cloud-core >= 0.24.0, < 0.25dev',
+    'google-cloud-core >= 0.24.0, < 0.26dev',
```

Comment:
Why this bump? It will be two generations of `google-cloud-core` and could be problematic?